cmd/config.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/redhat-developer/odo/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+// configurationCmd represents the app command
+var configurationCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Modifies configuration settings",
+	Long:  `Modifies odo specific configuration settings on config file.`,
+	Example: fmt.Sprintf("%s\n%s\n",
+		configurationGetCmd.Example,
+		configurationSetCmd.Example),
+	Aliases: []string{"configuration"},
+	// 'odo config' is the same as 'odo config get'
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) >= 1 && (args[0] != "get" || args[0] != "set") {
+			return fmt.Errorf("Unknown usage")
+		}
+		return nil
+	}, Run: func(cmd *cobra.Command, args []string) {
+		if len(args) > 0 && args[0] == "set" {
+			configurationSetCmd.Run(cmd, args)
+		} else if len(args) > 0 && args[0] == "get" {
+			configurationGetCmd.Run(cmd, args)
+		} else {
+			configurationGetCmd.Run(cmd, args)
+		}
+	},
+}
+
+var configurationSetCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Set a value in odo config file",
+	Long:  "Set an individual value in odo configuration file",
+	Example: `  # Set a configuration value
+   odo config set UpdateNotification false
+ 	`,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 2 {
+			return fmt.Errorf("Please provide parameter name and value")
+		} else if len(args) > 2 {
+			return fmt.Errorf("Only one value per parameter name is allowed")
+		} else {
+			return nil
+		}
+	}, Run: func(cmd *cobra.Command, args []string) {
+
+		switch args[1] {
```

Comment:
you can use strconv.ParseBool to avoid this switch case and make the code more robust.